.github/workflows/l2-contracts-ci.yaml

```diff
@@ -75,6 +75,7 @@ jobs:
         run: yarn lint:check
 
   test:
+    if: false
```

Comment:
This was moved to system contracts ci? Should it be removed if so?

Reply:
Yeah, just wanted to flag, that we actually don't run any tests for other l2 contracts in CI